topics/cdnjs/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-created_by: Ryan Kirkman, Thomas Davis, Peter Dave, Matt Cowley
+created_by: Matt Cowley, Sven Sauleau, Tyler Caslin, Drew Freyling, Alex, Ryan Kirkman, Thomas Davis
```

Comment:
the website states that Thomas Davis and Ryan Kirkman are the two founders. Perhaps the current version is incorrect as well. Thoughts on using this?
```suggestion
created_by: Ryan Kirkman and Thomas Davis
```

Reply:
👋🏼 Hi from cdnjs. Ryan & Thomas are indeed our founders, but they’re now rather inactive. The team listed on the website and here are the ones that now maintain and continue to create cdnjs. I guess this is a question of wether this field is for original creators (founded in 2011), or those who currently run it?

Reply:
The intention behind `created_by` is to define the original creators, see github.com/topics/js for a good example of that. I think we mistakenly included a wide net when this topic was first created. I'm happy to make those changes outside of this PR, or you can include them here.